package.json

```diff
@@ -23,7 +23,7 @@
   },
   "dependencies": {
     "@angular/animations": "~7.2.7",
-    "@angular/cdk": "7.3.2",
+    "@angular/cdk": "7.3.7",
```

Comment:
Is this a necessary change?

Reply:
@KevinCJH and I couldn't get the application to respond with the original `"@angular/cdk": "7.3.2"` but after triggering `npm update @angular/cdk` the `cdk` version was automatically updated in the `package.json` file. I've tried renaming back to `7.3.2` and it caused no issues except `IntelliJ` highlighting that the version of `cdk` installed in the app is newer than `7.3.2`.

Shall I just leave it as `7.3.2`?

Reply:
It is okay for now, lets keep the updated version